packages/widget-v2/src/state/swapExecutionPage.ts

```diff
@@ -21,7 +21,7 @@ type SwapExecutionState = {
   route?: RouteResponse;
   transactionDetailsArray: TransactionDetails[];
   transactionHistoryIndex: number;
-  overallStatus?: SimpleStatus;
+  overallStatus: SimpleStatus;
```

Comment:
I changed this so it can't be undefined, but instead returns "" empty string if it would other wise be undefined, it's kinda weird, but I mainly want to be able to use this:

```ts

["completed", "failed"].includes(overallStatus);
```
but we get type errors if overallStatus can be undefined